data/Rahul-Prasad-07.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "Rahul Prasad",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": " Blockchain Developer | **Open Source** Advocate | DevOps Developer",
+  "tags": ["Open Source","Blockchain", "DevOps", "Kubernetes", "Docker", "Node.js"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/rahultwte" },
+    { "icon": "FaGithub", "url": "https://github.com/Rahul-Prasad-07" },
+    { "icon": "website", "url": "https://dev.to/rahulprasad07" }
```

Comment:
This should be an icon from the icon sets we approve, you can search for them here https://linkfree.eddiehub.io/icons